ontotext-graphql-playground-component/src/mappers/graphiql-configuration.mapper.ts

```diff
@@ -0,0 +1,28 @@
+import {ExternalGraphqlPlaygroundConfiguration} from '../models/external-graphql-playground-configuration';
+import {GraphiQLProps} from '../graphiql-models/graphiql-props';
+import {FetcherConfiguration} from './fetcher-configuration.mapper';
+
+/**
+ * Mapper class for converting an external GraphQL playground configuration into a configuration compatible with GraphiQL.
+ */
+export class GraphiqlConfigurationMapper {
+  
+  /**
+   * Converts an external configuration into a GraphiQL configuration, which is used when initializing the GraphiQL component.
+   *
+   * @param {Object} configuration - The external configuration that configures the GraphiQL component.
+   */
+  static toGraphiQLConfiguration(configuration: ExternalGraphqlPlaygroundConfiguration): GraphiQLProps {
+    const graphiQLConfig: GraphiQLProps = {
+      defaultEditorToolsVisibility: false,
+      selectedLanguage: configuration.selectedLanguage || 'en',
+      defaultQuery: ' ',
+      translations: configuration.translations,
+      // @ts-ignore
```

Comment:
what are you ignoring here?

Reply:
React, GraphiQL and GraphiQLPluginExplorer are global variables that will be provided at runtime.